Lib/defconQt/glyphCollectionView.py

```diff
@@ -30,11 +30,18 @@
 
 
 def proceedWithDeletion(self, erase=False):
-    do = "clear" if not erase else "delete"
+    clear_header = QCoreApplication.translate(
+        "GlyphCollectionWidget", "Clear glyphs")
+    clear_text = QCoreApplication.translate(
+        "GlyphCollectionWidget", "Do you want to clear selected glyphs?")
+    erase_header = QCoreApplication.translate(
+        "GlyphCollectionWidget", "Delete glyphs")
+    erase_text = QCoreApplication.translate(
+        "GlyphCollectionWidget", "Do you want to delete selected glyphs?")
```

Comment:
tbh I think we could just make `proceedwithdeletion` a mthod of GlyphCollectionView and so use self.tr here.